man/tpm2_sign.1.md

```diff
@@ -39,6 +39,16 @@ data and validation shall indicate that hashed data did not start with
     Also, see section "Supported Hash Algorithms" for a list of supported hash
     algorithms.
 
+  * **-e**, **--signscheme**=_SIGNING\_SCHEME_:
```

Comment:
change the current `-s` to `-o` | `--out-sig` and use `-s` | `--sig-scheme` for this.